android/modules/ui/src/js/tabgroup.js

```diff
@@ -59,6 +59,18 @@ exports.bootstrap = function(Titanium) {
 		}
 	}
 
+	var _removeTab = TabGroup.prototype.removeTab;
+	TabGroup.prototype.removeTab = function(options) {
+		if (this.currentState != this.state.opened) {
+			var index = this._tabs.indexOf(options);
+				if (index > -1) {
+				  this._tabs.splice(index, 1);
+				}
```

Comment:
<!--
  1 failure:  android/modules/u...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: android/modules/ui/src/js/tabgroup.js;
  Line: 68;
-->
- :no_entry_sign: android/modules/ui/src/js/tabgroup.js line 68 – Expected indentation of 3 tabs but found 4. (indent)